server/testutil/testregistry/testregistry.go

```diff
@@ -133,17 +137,60 @@ func (r *Registry) PushNamedImage(t *testing.T, imageName string) (string, gcr.I
 	files := map[string][]byte{
 		"/tmp/" + imageName: []byte(imageName),
 	}
-	image, err := crane.Image(files)
+	image, err := imageFromFiles(files)
 	require.NoError(t, err)
 	return r.Push(t, image, imageName), image
 }
 
 func (r *Registry) PushNamedImageWithFiles(t *testing.T, imageName string, files map[string][]byte) (string, gcr.Image) {
-	image, err := crane.Image(files)
+	image, err := imageFromFiles(files)
 	require.NoError(t, err)
 	return r.Push(t, image, imageName), image
 }
 
+func imageFromFiles(files map[string][]byte) (gcr.Image, error) {
```

Comment:
maybe add a comment on why we don't use crane.Image here? Seems likely that someone might try to clean this up/undo the fix by replacing it with crane.Image